app/views/job_templates/_custom_tabs.html.erb

```diff
@@ -10,7 +10,8 @@
 
   <%= text_f f, :description_format,
              :help_inline => popover(_('Explanation'), _('Description template determines the job name once it is submitted. Input values can become part of the name
-                         if they are specified using interpolation syntax, e.g. %{fqdn} where fqdn is the name of interpolated input.')) %>
+                         if they are specified using interpolation syntax, e.g. %{fqdn} where fqdn is the name of interpolated input.
+                         An exception is %{job_name} which is always replaced by template name.')) %>
```

Comment:
oops, it's a invocation job name, I took this from another PR that was replacing job name with template name, will fix the help